templates/CRM/Admin/Page/Extensions/Main.tpl

```diff
@@ -12,7 +11,6 @@ Depends: CRM/common/enableDisableApi.tpl and CRM/common/jsortable.tpl
         <tr>
           <th>{ts}Name{/ts}</th>
           <th>{ts}Status{/ts}</th>
-          <th>{ts}Version{/ts}</th>
```

Comment:
is this the right header being removed? I though Type was going

Reply:
Fixed :man_facepalming: 
Thanks :)